x/testnet/command.go

```diff
@@ -0,0 +1,77 @@
+package testnet
```

Comment:
This should not be a module. Have it live under the parent somewhere or under client.

Reply:
hmm - def not in the repo root, this is utilizing all sorts of `x/` functionality, In the long run, nothing outside of `x/` should depend on anything inside `x/` (only the reverse should be true) it's actually causing a lot of dep issues for client generalization right now. 

I guess this _could_ maybe live within `genutil` but I'd want to keep everything totally separate which conflicts with the module subpackage structure in `genutil` so I'm apt to keep this as it's own module... why not? it seems to be a more or less totally standalone piece of functionality.

Reply:
@rigelrozanski I believe it should not be it's own standalone module -- this "module" is literally just a command implementation.

I'm relatively indifferent on where it actually _does_ live (eg. under `x/genutil/client/cli` or anywhere else), but I'm pretty certain this isn't the abstraction we want here.

Seems like the comprise (I hope) is to move this elsewhere under your discretion 🙏 

------

Also, wrt to the statement:

> In the long run, nothing outside of `x/` should depend on anything inside `x/` (only the reverse should be true) ...

Is this really true? In most cases, yes, I think we want to avoid these situations as much as possible, but I don't think this a dogma that want to shove down the throats of developers and users lol

Reply:
lol I wouldn't equate a dependency management strategies as dogma. Also this really only applies to core-devs who are modifying `cosmos/cosmos-sdk` (and for the record it should not be shoved down anyone's throat lol - only explained to a level where the dev understands and is in agreement) 

- we're unfortunately not following it right now - The hope is that any elements which a blockchain project may want to swap or switch up should be a module such that the project can simply redefine a particular module and _not have to fork the whole sdk_. Some functionality within root packages of the sdk (outside of the `cmd/` and `/x`) currently depend on `auth` module and other modules I believe, this would become problematic if a different auth module is desired to be used, hence the whole sdk would need to be forked under that situation. 

The ramification of other projects forking the sdk to achieve desired functionality instead of importing the sdk and using modules is it becomes harder for them to contribute back to the `cosmos/cosmos-sdk`. 

so yeah I def standby this idea that `x/` should be self contained and only imported by `cmd/`

______________________

I guess I'll see about adding this to `genutil` although it may cause dependancy problems, it's kind of separate and builds on both `genaccounts` and `genutil`... I'll try some options out though